config/locales/en.yml

```diff
@@ -265,14 +275,14 @@ en:
   episode_player:
     form:
       enclosure_not_ready: No Media Files Uploaded
-      help:
+      help: &player_help
         copy: Copy the code for one of the following players, and place it on your webpage.
         types:
           card: Something about a responsive card
-          card_fixed: Fixed width card something blah
+          fixed_card: Fixed width card something blah
```

Comment:
this was backwards so fixed it here.

Reply:
Ha - and also @brandonhundt you may want to fix these pieces of help text.

Reply:
```
card: Default view of the PRX Player, will adjust to be responsive to the size of the user's screen and the website it is embedded on. Must always be 200px tall.
card_fixed: The player style that shows off your podcast's artwork. The size will NOT adjust to the user's screensize. Must always be 500px wide by 800px tall.
fixed_card: The player style that shows off your podcast's artwork and will adjust to be responsive to the size of the user's screen and the website it is embedded on. Must always be 800px tall.
```